openmessaging-format.md

```diff
@@ -0,0 +1,130 @@
+# OpenMessaging Event Format for CloudEvents - Version 0.1
+
+## Abstract
+
+The OpenMessaging Format for CloudEvents defines how events are expressed in
+OpenMessaging.
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Envelope](#3-envelope)
+4. [References](#4-references)
+
+## 1. Introduction
+
+[CloudEvents][CE] is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification defines how
+the elements defined in the CloudEvents specification are to be represented in
+the [OpenMessaging][OpenMessaging].
+
+The [Attributes](#2-attributes) section describes the naming conventions and
+data type mappings for CloudEvents attributes.
+
+The [Envelope](#3-envelope) section defines a OpenMessaging container for CloudEvents
+attributes and an associated media type.
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][RFC2119].
+
+## 2. Attributes
+
+This section defines how CloudEvents attributes are mapped to OpenMessaging. This
+specification does not explicitly map each attribute, but provides a generic
+mapping model that applies to all current and future CloudEvents attributes.
+
+### 2.1. Base Type System
+
+The core [CloudEvents specification][CE] defines a minimal abstract type
+system, which this mapping leans on.
+
+### 2.2. Type System Mapping
```

Comment:
This feels like a duplicate of what's in the CE specs already and so I'd prefer to just reference those other docs instead of duplicate it here and have to worry about keeping them in-sync.

Reply:
In here，we just want to show what we lean on, and we will remove the definitive description of CE